pydantic/config.py

```diff
@@ -36,6 +36,12 @@ class Extra(str, Enum):
     forbid = 'forbid'
 
 
+class Copy(str, Enum):
```

Comment:
Breast to use a `Literal` not an enum. Also empty string should not be an option.